src/utils/featheryClient/index.ts

```diff
@@ -492,18 +518,30 @@ export default class FeatheryClient extends IntegrationClient {
     };
     if (collaboratorId) data.collaborator_user = collaboratorId;
     const options = {
-      headers: { 'Content-Type': 'application/json' },
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Token ${sdkKey}`
+      },
       method: 'POST',
       body: JSON.stringify(data)
     };
 
-    // Ensure events complete before user exits page. Submit and load event of
-    // next step must happen after the previous step is done submitting
-    return await this.submitQueue.then(() =>
-      wrapUnload(() =>
-        this.draft ? Promise.resolve() : this._fetch(url, options)
-      )
-    );
+    const request = new Request(url, options);
+    let eventType = '';
+    let eventStep = '';
+    if (eventData.event === 'complete') {
+      eventType = 'completeStep';
```

Comment:
does the eventType need to be passed to indexdb? i'm not sure we need to run the complete and load events separately as long as we just initiate the requests in order